src/FSharpPlus/Monoids.fs

```diff
@@ -1,11 +1,14 @@
 ﻿namespace FSharpPlus.Data
 
 open FSharpPlus.Operators
+open System.ComponentModel
 
 /// The dual of a monoid, obtained by swapping the arguments of append.
 [<Struct>]
 type Dual<'t> = Dual of 't with
+    [<EditorBrowsable(EditorBrowsableState.Never)>]
     static member inline get_Zero () = Dual (getZero ())        : Dual<'T>
+    [<EditorBrowsable(EditorBrowsableState.Never)>]
```

Comment:
I wouldn't hide operators, we're expecting them to be used, even if it's not through F#+.